packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[].name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.name"
+									}
+								}
+							]
+						},
+						"in": {
+							"__regexp": "^(path|query|header|cookie)$",
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.in"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.in"
+									}
+								}
+							]
+						},
+						"schema": {
+							"__conditions": [
+								{
+									"__if": {
+										"default": ".*"
+									},
+									"__then": {
+										"default": {
+											"__actions": [
+												{
+													"__copy": {
+														"__target": "endpoints[_].parameters[_].default"
+													}
+												}
+											]
+										}
+									}
+								}
+							],
+							"__ignore": {}
+						},
+						"__ignore": {}
+					}
+				},
+				"__ignore": {}
+			}
+		}
+	},
+	"components": {
+		"schemas": {
+			"securitySchemes": {
+				"__objectItem": {
+					"type": {
+						"__regexp": "^(apiKey|http)$"
+					},
+					"in": {
+						"__regexp": "^(query|header|cookie)$"
+					},
+					"__conditions": [
+						{
+							"__if": {
+								"type": "^(apiKey|http)$"
+							},
+							"__then": {
+								"type": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.type"
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"in": "^(query|header|cookie)$"
+							},
+							"__then": {
+								"in": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.in"
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"type": "^apiKey$"
+							},
+							"__then": {
+								"name": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.name"
+											}
+										},
+										{
+											"__insert": {
+												"__target": "apiSpecifications.security.{{3}}",
+												"__value": []
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"type": "^http$"
+							},
+							"__then": {
+								"scheme": {
+									"__regexp": "^(Basic|Bearer)$",
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.scheme"
+											}
+										},
+										{
+											"__insert": {
+												"__target": "apiSpecifications.security.{{3}}",
+												"__value": []
+											}
```

Comment:
Same as comment: https://github.com/api3dao/airnode/pull/195/files#r578923631

Reply:
Moved in 8e97859